test/tracing/timetodisplay.test.tsx

```diff
@@ -373,6 +379,67 @@ describe('TimeToDisplay', () => {
     expect(spanToJSON(initialDisplaySpan!).timestamp).toEqual(initialDisplayEndTimestampMs / 1_000);
     expect(spanToJSON(fullDisplaySpan!).timestamp).toEqual(fullDisplayEndTimestampMs / 1_000);
   });
+
+  test('should not log a warning if native component exists and not in new architecture', async () => {
+
+    (isTurboModuleEnabled as jest.Mock).mockReturnValue(false);
+
+    const [testSpan, activeSpan] = startSpanManual(
+      {
+        name: 'Root Manual Span',
+        startTime: secondAgoTimestampMs(),
+      },
+      (activeSpan: Span | undefined) => {
+        const testSpan = startTimeToInitialDisplaySpan();
+        TestRenderer.create(<TimeToInitialDisplay record={true} />);
+
+        emitNativeInitialDisplayEvent();
+
+        activeSpan?.end();
+
+        return [testSpan, activeSpan];
+      },
+    );
```

Comment:
Since we are only testing the warning, lest only render the component, the spans are not relevant and they are tested in the other tests in the file.

```suggestion
TestRenderer.create(<TimeToInitialDisplay record={true} />);
```